pinot-core/src/main/java/org/apache/pinot/core/data/manager/InstanceDataManager.java

```diff
@@ -134,7 +134,8 @@ void reloadAllSegments(String tableNameWithType)
   ZkHelixPropertyStore<ZNRecord> getPropertyStore();
 
   /**
-   * Returns the segment uploader, which uploads a llc segment to the destination place and returns the url of uploaded segment file. Servers utilize segment uploader to upload llc segment to segment store.
+   * Returns the segment uploader, which uploads a llc segment to the destination place and returns the url of uploaded segment file.
```

Comment:
This seems still over 120 characters. Shall we try to make all of them under 120 characters if possible? Same for other files

Reply:
We have a checkstyle hard failure limit of 160. 
For now, I break the line at 140, tried with 120, and it makes many expressions look a bit ugly with intentions.

Reply:
Suggest keeping the 120 because everyone has been following the code format